client/my-sites/stats/stats-module/index.jsx

```diff
@@ -197,11 +197,10 @@ class StatsModule extends Component {
 					{ summary && 'countryviews' === path && (
 						<UpgradeNudge
 							title={ translate( 'Add Google Analytics' ) }
-							message={ translate(
-								'Upgrade to a Business Plan for Google Analytics integration.'
-							) }
+							message={ translate( 'Upgrade to a Premium Plan for Google Analytics integration.' ) }
```

Comment:
I'm intrigued as to why this nudge is only on the countries view. Seems like we should put it on the others depending on how often people click it.